common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -130,4 +133,44 @@ public static Optional<ChatFormatting> getChatFormatting(int textColor) {
                 .filter(c -> c.isColor() && textColor == c.getColor())
                 .findFirst();
     }
+
+    public static List<Component> stripDuplicateBlank(List<Component> lore) {
```

Comment:
Just curious. When do we get duplicate blank lines? Is that in lore coming from Wynncraft, or after we've processed it? (And if it's the latter, is it not better to avoid inserting duplicated blank lines in the first place?)

Reply:
it's called in both contexts, actually. i'm pretty sure it's to deal with the unpredictable nature of wynncraft tooltip formats, but i'm not entirely sure if it's necessary - this logic was mostly copied over from the existing tooltip code, since it was already functional

Reply:
Ah, I see. (Hard to understand what is new code and what is moved.) Well, defensive programming is good, I guess.